ERR! OMG CMake executable is not found.

during npm install -g @devicefarmer/stf I have an error
prebuild-install WARN install No prebuilt binaries found (target=14.13.1 runtime=node arch=x64 libc= platform=darwin)
[
  '/usr/local/Cellar/node/14.13.1_1/bin/node',
  '/usr/local/lib/node_modules/@devicefarmer/stf/node_modules/.bin/cmake-js',
  'compile',
  '--target',
  'jpegturbo'
]
info TOOL Using Unix Makefiles generator.
info TOOL Building only the jpegturbo target, as specified from the command line.
info REP Build has been failed, trying to do a full rebuild.
ERR! OMG CMake executable is not found. Please use your system's package manager to install it, or you can get installers from there: http://cmake.org.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @julusian/jpeg-turbo@0.5.6 install: `prebuild-install || cmake-js compile --target jpegturbo`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @julusian/jpeg-turbo@0.5.6 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
pls help

--- a/README.md
+++ b/README.md
@@ -109,6 +109,7 @@ As the product has evolved from an internal tool running in our internal network
 * [Node.js](https://nodejs.org/) 8.x **required** (some dependencies don't support newer versions)
 * [ADB](http://developer.android.com/tools/help/adb.html) properly set up
 * [RethinkDB](http://rethinkdb.com/) >= 2.2
+* [CMake](https://cmake.org/) >= 3.9 (for [node-jpeg-turbo](https://github.com/julusian/node-jpeg-turbo#readme))
 * [GraphicsMagick](http://www.graphicsmagick.org/) (for resizing screenshots)
 * [ZeroMQ](http://zeromq.org/) libraries installed
 * [Protocol Buffers](https://github.com/google/protobuf) libraries installed
@@ -120,7 +121,7 @@ Note that you need these dependencies even if you've installed STF directly from
 On Mac OS, you can use [homebrew](http://brew.sh/) to install most of the dependencies:
 
 ```bash
-brew install rethinkdb graphicsmagick zeromq protobuf yasm pkg-config
+brew install rethinkdb graphicsmagick zeromq protobuf yasm pkg-config cmake
 ```
 
 On Windows you're on your own. In theory you might be able to get STF installed via [Cygwin](https://www.cygwin.com/) or similar, but we've never tried. In principle we will not provide any Windows installation support, but please do send a documentation pull request if you figure out what to do.

